app/src/androidTest/java/de/westnordost/streetcomplete/quests/existence/CheckExistenceTitleTest.kt

```diff
@@ -33,56 +33,68 @@ class CheckExistenceTitleTest {
 
     // https://github.com/streetcomplete/StreetComplete/issues/2512
     @Test fun issue2512() {
-        val newspaperVendingMachineWithName = getQuestTitle(mapOf(
-            "amenity" to "vending_machine",
-            "vending" to "newspapers",
-            "name" to "Bild",
-        ))
+        val newspaperVendingMachineWithName = getQuestTitle(
+            mapOf(
+                "amenity" to "vending_machine",
+                "vending" to "newspapers",
+                "name" to "Bild",
+            )
+        )
         assertEquals(newspaperVendingMachineWithName, "Is Bild (Newspaper Vending Machine) still here?")
 
-        val newspaperVendingMachineWithBrand = getQuestTitle(mapOf(
-            "amenity" to "vending_machine",
-            "vending" to "newspapers",
-            "brand" to "Abendzeitung",
-        ))
+        val newspaperVendingMachineWithBrand = getQuestTitle(
+            mapOf(
+                "amenity" to "vending_machine",
+                "vending" to "newspapers",
+                "brand" to "Abendzeitung",
+            )
+        )
         assertEquals(newspaperVendingMachineWithBrand, "Is Abendzeitung (Newspaper Vending Machine) still here?")
     }
 
     // https://github.com/streetcomplete/StreetComplete/issues/2640
     @Test fun issue2640() {
-        val postBox = getQuestTitle(mapOf(
-            "amenity" to "post_box",
-            "brand" to "Deutsche Post",
-            "operator" to "Deutsche Post AG",
-            "ref" to "Hauptsmoorstr. 101, 96052 Bamberg",
-        ))
+        val postBox = getQuestTitle(
+            mapOf(
+                "amenity" to "post_box",
+                "brand" to "Deutsche Post",
+                "operator" to "Deutsche Post AG",
+                "ref" to "Hauptsmoorstr. 101, 96052 Bamberg",
+            )
+        )
```

Comment:
Which rule is this? This was deliberate, to keep the number of indentations as low as possible because I think it does improve redability.